lib/std/mem.zig

```diff
@@ -150,9 +220,21 @@ pub const Allocator = struct {
         /// null means naturally aligned
         comptime alignment: ?u29,
         n: usize,
+    ) Error![]align(alignment orelse @alignOf(T)) T {
+        return self.alignedAlloc2(T, alignment, n, .exact);
+    }
+
+    const Exact = enum {exact,atLeast};
```

Comment:
I didn't worry about style conventions stuff elsewhere, but for APIs it matters since to fix it would be a breaking API change:

```suggestion
    const Exact = enum {exact,at_least};
```